latch_cli/main.py

```diff
@@ -228,6 +228,14 @@ def register(
     type=str,
     help="Image to use for develop session.",
 )
+@click.option(
+    "-s",
+    "--snakemake",
```

Comment:
do you not need to pass in the snakefile path? how are you handling that

Reply:
The Snakefile is passed in at registration time and hardcoded into the generated JIT code (and it should be copied into the Docker image). So as long as the path to the Snakefile doesn't change, the JIT code should be able to find the Snakefile

Reply:
right its already in the image in a preset path, maketh the sense